data/cybox/nozomi/nozomi_alerts_22022024.json

```diff
@@ -0,0 +1,563 @@
+                    "parent_directory_ref": "6"
+                },
+                "8": {
+                    "type": "process",
+                    "command_line": "C:\\Windows\\System32\\cmd.exe",
+                    "binary_ref": "7"
+                },
+                "9": {
+                    "type": "directory",
+                    "path": "C:\\UDI automation\\venv\\Scripts"
+                },
+                "10": {
+                    "type": "file",
+                    "name": "pip.exe",
+                    "parent_directory_ref": "9",
+                    "hashes": {
+                        "SHA-256": "4C5C1FDC22565CE3B9C881A322CF3F24E3CB31DBEFD8E0271EXXXXX"
+                    }
+                },
+                "11": {
+                    "type": "file",
+                    "hashes": {
+                        "SHA-256": "4C5C1FDC22565CE3B9C881A322CF3F24E3CB31DBEFD8E0271EXXXXX"
+                    }
+                }
+            },
+            "first_observed": "2023-12-12T07:10:57.012Z",
+            "last_observed": "2023-12-12T07:10:57.012Z",
+            "number_observed": 1
+        },
+        {
+            "id": "observed-data--77e9a3d1-5afc-48dd-ad03-7c578ede8ff7",
+            "type": "observed-data",
+            "created_by_ref": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "created": "2024-02-22T06:51:20.238Z",
+            "modified": "2024-02-22T06:51:20.238Z",
+            "objects": {
+                "0": {
+                    "type": "x-ibm-finding",
+                    "alert_id": "69f609c3-6564-4d16-aa8e-1dc8746f41e1",
+                    "x_is_acknowledged": false,
+                    "severity": 90,
+                    "time_observed": "2023-12-14T09:14:39.802Z",
+                    "dst_ip_ref": "1",
+                    "src_ip_ref": "3",
+                    "x_alert_status": "open",
+                    "name": "Bad IP reputation (new node)",
+                    "x_cause": "An IP address which is known to be malicious has appeared in the network. Likely some kind of malware is attempting to perform malicious activity",
+                    "ioc_refs": [
+                        "8"
+                    ],
+                    "x_solution": "Get in touch with the local site to make sure that any kind of malware is removed from the involved Assets.",
+                    "ttp_tagging_refs": [
+                        "9",
+                        "10"
+                    ],
+                    "description": "New IP node 109.205.213.34 has appeared, that is known to have bad reputation [RDP bruteforce]",
+                    "x_is_incident_alert": false,
+                    "x_is_cybersecurity_alert": true,
+                    "x_threat_name": "RDP bruteforce",
+                    "finding_type": "threat",
+                    "start": "2023-12-14T09:14:39.802Z",
+                    "x_sensor_host": "EC2AMAZ-XXXXX",
```

Comment:
This seems to be a partially obscured real hostname. When obscuring values, it's best to remove any references to the real data and use a defined replacement. For example "Hostname" or "Host" might work better.